mission_control/navigator_missions/nav_missions/find_the_break.py

```diff
@@ -1,12 +1,79 @@
 #!/usr/bin/env python
 import txros
 import numpy as np
-from navigator_tools import fprint
+import sys
+import navigator_tools as nt
+import sys
+import itertools
+from navigator_tools import fprint, DBHelper
+from navigator_testing_suite import FindTheBreakTestPerception
+from navigator_find_the_break import FindTheBreakPerception
 
 
 @txros.util.cancellableInlineCallbacks
 def main(navigator, **kwargs):
-    fprint("STARTING FIND THE BREAK, GETTING A BOUY AS A TEST", msg_color="green")
+    # mission params
+    meters_between_buoys = 20
+    radius = 40
+
+    # var initialization
+    nh = navigator.nh
+    test = False
```

Comment:
For dictionaries, you can do `test = kwargs.get("test", False)` to get it if it's in the dictionary and default to False if it's not